components/Project/EvaluationDisplay.tsx

```diff
@@ -0,0 +1,129 @@
+import { RequirementEvaluation } from '@idea2app/data-server';
+import { Box, Typography } from '@mui/material';
+import { FC, useContext } from 'react';
+
+import { I18nContext } from '../../models/Translation';
+
+export const EvaluationDisplay: FC<RequirementEvaluation> = ({
```

Comment:
```suggestion
export const EvaluationDisplay: FC<RequirementEvaluation> = observer(({
```
MobX-i18n 需要让组件变成观察者。